.github/workflows/process_external_prs.yml

```diff
@@ -0,0 +1,29 @@
+name: Process External PRs
+
+on:
+  pull_request_target: # Please read https://securitylab.github.com/research/github-actions-preventing-pwn-requests/ before using
+     types: [opened, ready_for_review, closed] # 'closed' necessary for 'merged-comment' below
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  add-external-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add label
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: O-external
+
+  assign-reviewer:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post comment
+        uses: hkusu/review-assign-action@v1
+        with:
+          assignees: ${{ github.actor }} # assign pull request author
+          reviewers: docs-ext-pr-reviewers # assign docs-ext-pr-reviewers as reviewer
+          ready-comment: ":tada: Thanks for your Pull Request! We appreciate your help improving our documentation!<br><br>:memo: Please take a moment to check the following:<ul><li>Have you signed our contributor agreement? If not, look for a comment from **cockroach-teamcity** with instructions. We cannot review your submission until you've signed this agreement.</li><li>Does your submission adhere to our [Style Guide](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md)? Please edit your PR to meet these guidelines if necessary: our docs follow our style guide to maintain consistency, readability, and clarity throughout.</li></ul><br>:writing_hand: We'll respond here as soon as we can. Thanks!"
```

Comment:
```suggestion
          ready-comment: ":tada: Thanks for your pull request! We appreciate your help improving our documentation!<br><br>:memo: Please take a moment to check the following:<ul><li>Have you signed our contributor agreement? If not, look for a comment from **cockroach-teamcity** with instructions. We cannot review your submission until you've signed this agreement.</li><li>Does your submission adhere to our [Style Guide](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md)? Please edit your PR to meet these guidelines if necessary: our docs follow our style guide to maintain consistency, readability, and clarity throughout.</li></ul><br>:writing_hand: We'll respond here as soon as we can. Thanks!"
```

Reply:
See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests for spelling.